packages/forms/src/model.ts

```diff
@@ -69,18 +69,32 @@ function _find(control: AbstractControl, path: Array<string|number>|string, deli
   return controlToFind;
 }
 
+function pickValidators(validatorOrOpts?: ValidatorFn|ValidatorFn[]|AbstractControlOptions|
+                        null): ValidatorFn|ValidatorFn[]|null {
+  const _validator = isOptionsObj(validatorOrOpts) ? validatorOrOpts.validators : validatorOrOpts;
+  return _validator || null;
```

Comment:
NIT: I don't know if the minifier would remove it for us but the extra `_validator` variable seems unnecessary:
Same for `pickAsyncValidators()`.

```suggestion
  return (isOptionsObj(validatorOrOpts) ? validatorOrOpts.validators : validatorOrOpts) || null;
```